client/extensions/woocommerce/components/address-view/index.js

```diff
@@ -116,19 +133,47 @@ class AddressView extends Component {
 		);
 	};
 
+	onClickShowAddressLine2 = event => {
+		event.preventDefault();
+		this.setState( { showAddressLine2: true } );
+	};
+
+	renderAddressLine2 = () => {
+		const { address, onChange, translate } = this.props;
+		const { street2 } = address;
+		const { showAddressLine2 } = this.state;
+
+		if ( showAddressLine2 ) {
+			return (
+				<FormFieldSet>
+					<FormTextInput name="street2" onChange={ onChange } value={ street2 } />
+				</FormFieldSet>
+			);
+		}
+
+		return (
+			<Button
+				borderless
+				className="address-view__show-line-2"
+				onClick={ this.onClickShowAddressLine2 }
+			>
+				<Gridicon icon="plus-small" />
+				{ translate( 'Add Address Line 2' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **44** times:
`translate( '+ Add Address Line 2' )` *ES Score: 14*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/fix/16433-address-line-two-on-demand)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).